pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <groupId>com.facebook.presto</groupId>
     <artifactId>presto-root</artifactId>
-    <version>0.149-tw-0.27</version>
+    <version>0.149-tw-0.28</version>
```

Comment:
True. Should not include the reversion. 
Pushed with a correction of the typo. Will remove it.